app/src/main/res/drawable/clickable_primary.xml

```diff
@@ -3,7 +3,7 @@
 
     <item android:state_pressed="false">
         <shape android:shape="rectangle">
-            <solid android:color="?colorPrimary" />
+            <solid android:color="@color/theme500" />
```

Comment:
:pensive: Unfortunately, the change seems not fixing the issue. The same error: `Can't convert to color: type=0x2 error`.